core/src/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -390,7 +387,7 @@ private SecRecord CreateRecord(string account, string service, string generic, i
                 Synchronizable = _defaultSyncSetting,
             };
         }
-        
+
         private SecStatusCode Remove(string account, string service, int type)
```

Comment:
nit: seems like a number of lines in a few places like here.

Reply:
I think I removed all the extra lines. #!*